### ISSUE TITLE
in der user-info-tags werden nicht alle Tags berücksichtigt

### ISSUE THREAD

**Issue by MatthiasApsel** [ckruse/cforum#340]
Bei mir zum Beispiel die aus dem internen Forum nicht. Ist das so gewollt?

Frage: Was ist, wenn es in verschiedenen Foren gleichbenannte Tags gibt?

**Comment by ckruse** [comment]
Als anonymer User: ja. Als Admin-User: nein.

Antwort: dann tauchen sie mehrfach auf :)

**Comment by ckruse** [comment]
Das Problem ist, dass bei verschobenen Einträgen die Tags immer noch zum anderen Forum gehören. Hm. Im Grunde müsste ich die Tags in das neue Forum kopieren...

**Comment by MatthiasApsel** [comment]
Dann ist das ein Fehler. Als admin taucht das Tag Moderation in meiner Seite nicht auf. Eigentlich sollten alle Foren berücksichtigt werden. Oder ist da ein Limit drauf? Werden nur die 10 häufigsten Tags gezeigt? War das schon immer so? Fände ich an der Stelle nicht gut.

Das mehrfach kann ich bestätigen. ;-)

**Comment by MatthiasApsel** [comment]
Zu diesem Forum gibt es in allen drei Foren.

**Comment by ckruse** [comment]
Ok, das Problem ist ein anderes: ich benutze ein `LIMIT 10` um zu vermeiden, dass die Tag-Wolke zu sehr wächst. Das sorgt dafür, dass die aus Intern nicht drin sind.

Eine mögliche Lösung wäre es, dass `LIMIT` wegzulassen, dann müsste man aber das Layout anpassen.

**Comment by MatthiasApsel** [comment]
Ja, das habe ich schon mal in meiner Test-installation probiert. Sieht tatsächlich gruselig aus. Ich würds erstmal so lassen, eventuell einen Link "zeige alle" oder so.

**Comment by ckruse** [comment]
Dann aber in der 4.2 ;-)
